pkg/nodeps/ddev_environment.go

```diff
@@ -0,0 +1,28 @@
+package nodeps
```

Comment:
```suggestion
package environment
```
What about a new package here?

Reply:
Done